src/utils/errors/contentTypeError.ts

```diff
@@ -0,0 +1,31 @@
+import HttpStatus from '@xpring-eng/http-status'
+
+import PayIDError from './payIdError'
+
+/**
+ * An enum containing the different kinds of Media Types (but also sometimes called "Content Types") used by the PayID server.
+ */
+export enum MediaType {
+  ApplicationJson = 'application/json',
+  ApplicationMergePatchJson = 'application/merge-patch+json',
+}
+
+/**
+ * A custom error type to organize logic around 415 - Unsupported Media Type errors.
+ * Https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/415.
+ */
+export default class ContentTypeError extends PayIDError {
+  public readonly kind: string
+
+  /**
+   * The constructor for new ContentTypeError.
+   *
+   * @param mediaType - The Media Type (also called Content Type) required by the request.
+   */
+  public constructor(mediaType: MediaType) {
```

Comment:
Can we just make this:

```suggestion
  public constructor(mediaType: 'application/json' | 'application/merge-patch+json') {
```

Reply:
I updated the code to reflect your suggestion.